Shared/Prefs.swift

```diff
@@ -188,4 +190,12 @@ open class MockProfilePrefs: Prefs {
         let keysToDelete: [String] = dictionary.keys.filter { $0.hasPrefix(self.prefix) }
         things.removeObjects(forKeys: keysToDelete)
     }
+
+    open func addListener(_ defaultName: String, callback: @escaping () -> Void) {
+
```

Comment:
What this empty functions should do?

Reply:
It is meant to be called whenever the "preference" changes. Take a look on the usage in `BrowerCliqz`